api/app/logs/config.py

```diff
@@ -23,10 +24,6 @@
         },
     },
     'handlers': {
```

Comment:
Let's talk with the VNG about removing Sentry and how this will impact installations at other Municipalities

Reply:
sentry is added back in and is disabled automaticly when azure is enabled